src/sentry/discover/arithmetic.py

```diff
@@ -20,15 +22,37 @@ class ArithmeticParseError(ArithmeticError):
     pass
 
 
+class ArithmeticValidationError(ArithmeticError):
+    """ The math itself isn't valid """
+
+    pass
+
+
+OperationSideType = Union["Operation", float]
```

Comment:
I might be missing something but this `Union` should include field names right? Since `transaction.duration` is neither an `Operation` nor a `float`